gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -588,6 +761,10 @@ func (p *glusterfileProvisioner) parseClassParameters(params map[string]string,
 	parseVolumeType := ""
 	parseVolumeOptions := ""
 	parseVolumeNamePrefix := ""
+	parseThinPoolSnapFactor := ""
+
+	//thin pool snap factor default to 1.0
+	cfg.thinPoolSnapFactor = float32(1.0)
```

Comment:
The default should be set as a constant somewhere.  You can document what the constant is for and then you would not need this comment here.